core/const.py

```diff
@@ -797,3 +797,62 @@ class ITLRegion(StrEnum):
     "Pre-Payment)",
     "Towns Fund RDEL Payment which is being utilised on TF project related activity",
 ]
+
+TABLE_AND_COLUMN_TO_ORIGINAL_COLUMN_LETTER = {
+    "Place Details": {"Question": "C", "Indicator": "D", "Answer": "E"},
+    "Project Details": {
+        "Project name": "E",
+        "Primary Intervention Theme": "F",
+        "Single or Multiple Locations": "G",
+        "Locations": "H/K",
+        "Postcodes": "H/K",
+        "Lat/Long": "I/L",
```

Comment:
I feel like this might breed some confusion as when I first read "H/K" I read it as "H-K", but it's fairly minor. We can leave it as is for now and get some design input.